src/test/java/com/exasol/adapter/dialects/exasol/ExasolSqlDialectIT.java

```diff
@@ -51,52 +65,59 @@
     private static final String TABLE_MIXED_CASE = "Table_Mixed_Case";
     private static final String VIRTUAL_SCHEMA_EXA_MIXED_CASE = "VIRTUAL_SCHEMA_EXA_Mixed_Case";
 
+    public static final String DEBUGGER_PORT = "8000";
+    public static final String EXASOL_DIALECT = "EXASOL";
     @Container
     private static final ExasolContainer<? extends ExasolContainer<?>> container = new ExasolContainer<>(
-            ExasolContainerConstants.EXASOL_DOCKER_IMAGE_REFERENCE) //
-                    .withLogConsumer(new Slf4jLogConsumer(LOGGER));
+            ExasolContainerConstants.EXASOL_DOCKER_IMAGE_REFERENCE).withLogConsumer(new Slf4jLogConsumer(LOGGER))
+                    .withReuse(true);
```

Comment:
Do we really need to re-use it here? We only have one test as far as I remember.

Reply:
It does not reuse the container intil you enable it in your llocal `~/.testconatiners.properties` file. In the CI it is never used. It makes testign just much faster, since it reuses the container between multiple runs.

Reply:
Then it's okay to have it here, but the method's name is very misleading. I would name it `possibleToReuse` or something like that. Now it sounds like it's always activated. But it's another topic

Reply:
I absolutely agree. But since it's the official testconatiners API so probably we have no chance to change it.